packages/grafana-llm-app/pkg/plugin/health.go

```diff
@@ -113,6 +126,14 @@ func (a *App) openAIHealth(ctx context.Context, req *backend.CheckHealthRequest)
 		d.Error = "No functioning models are available"
 	}
 
+	if d.Configured {
+		err := a.testOpenAIAssistant(ctx)
+		if err == nil {
+			d.Assistant.OK = true
+			d.Assistant.Error = ""
+		}
```

Comment:
Perhaps we should append the error to the `d.Assistant.Error` if it's non-nil, to make it easier for users to debug why the assistant isn't available.

Reply:
Good point, will do.